app/services/invite_provider_user.rb

```diff
@@ -42,6 +43,14 @@ def send_welcome_email
     ProviderMailer.account_created(@provider_user).deliver_later
   end
 
+  def send_slack_notification
+    providers = @provider_user.providers.map(&:name).to_sentence
+    message = "#{@provider_user.first_name} has been invited to join #{providers}"
```

Comment:
Do we store who invited each user? The inviter could be as important as the invitee when tracking usage.

Reply:
The audit trail seems to capture enough information, including whether the inviter is a support or provider user.